### ISSUE TITLE
[ui_next] Select All checkbox is out of alignment on lists when the Advanced Search selector is enabled

### ISSUE THREAD

**Issue by unlikelyzero** [ansible/awx#7825]
##### ISSUE TYPE
 - Bug Report

##### SUMMARY
Select All checkbox is out of alignment on lists when the Advanced Search selector is enabled.

##### ENVIRONMENT
* AWX version: ui_next

##### STEPS TO REPRODUCE
1. Navigate to a list (like Organizations)
2. Select the 'Advanced Search' Dropdown element.
Note: Select All checkbox is out of alignment but is still functional

##### EXPECTED RESULTS
I would expect the checkbox to be centered in the row

##### ACTUAL RESULTS

Select All checkbox is pushed up and left of center.

##### ADDITIONAL INFORMATION

From https://github.com/ansible/awx/pull/7739

but the spacing gets bad for middle-width views:

![Screen Shot 2020-07-28 at 3 31 30 PM](https://user-images.githubusercontent.com/1342624/88712259-6e03ff80-d0e7-11ea-8bae-a0281e4866bf.png)

Did figure out a workaround for this (https://stackoverflow.com/questions/35578404/giving-wrapped-flexbox-items-vertical-spacing/44523788) but I'm not adding it now in hopes it is fixed on the PF side.  @marshmalien made the discovery that flexbox will allow for a wrapping row gap (hopefully common place soon!) which would solve the problems:

https://developer.mozilla.org/en-US/docs/Web/CSS/gap#Browser_compatibility

The hack that works is:

```
.pf-c-toolbar__content-section {
  margin-top: -20px;
}
.pf-c-toolbar__content-section > * {
  margin-top: 20px;
}
```

**Comment by unlikelyzero** [comment]
Note: there is no associated PF issue filed. @trahman73 should I file one?

**Comment by trahman73** [comment]
@unlikelyzero Yes please! I know they are aware of the issue as we brought it up last week, but it would be nice to track it.

**Comment by unlikelyzero** [comment]
@trahman73 🔗  https://github.com/patternfly/patternfly/issues/3344

**Comment by AlexSCorey** [comment]
@unlikelyzero this is still a valid issue, however the screen shot above is not quite accurate any longer.  The buttons shown above are not buttons anymore, but instead are drop down items.  Adding a screen show for clarity.
![Screen Shot 2020-11-02 at 11 15 57 AM](https://user-images.githubusercontent.com/39280967/97891459-cce79f00-1cfc-11eb-8a9b-e9a4ef657984.png)

**Comment by marshmalien** [comment]
Status update - There is a fix pending in PatternFly Core:
Issue: patternfly/patternfly#3348
PR: patternfly/patternfly#3629

**Comment by keithjgrant** [comment]
Our mockups show a more advanced dropdown with something like "1 selected" — is this something to pursue while addressing this fix? 

![Screen Shot 2020-12-09 at 9 58 47 AM](https://user-images.githubusercontent.com/410794/101668353-4b72e300-3a05-11eb-9239-6f2ac86529aa.png)

**Comment by trahman73** [comment]
Here are the PF guidelines for the bulk select we want to use. (Whenever we get to this...)
https://www.patternfly.org/v4/guidelines/bulk-selection

**Comment by unlikelyzero** [comment]
still accurate
